bin/pre-release.sh

```diff
@@ -1,22 +1,37 @@
 #!/bin/sh
 set -e
 
-npm run test
-npm run build:package
-npm run build:dist
+echo "Starting a pre-release..."
+echo " "
+echo "This will:"
+echo "- run the test suite"
+echo "- build GOV.UK Frontend into the 'package/' directory"
+echo "- build GOV.UK Frontend into the 'dist/' directory"
+echo "- commit all changes and push the branch to remote"
+echo " "
 
-ALL_PACKAGE_VERSION=$(node -p "require('./package/package.json').version")
-TAG="v$ALL_PACKAGE_VERSION"
-CURRENT_BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
+read -r -p "Do you want to continue? [y/N] " continue_prompt
 
-if [ $(git tag -l "$TAG") ]; then
-    echo "⚠️ Git tag already exists."
-    exit 1;
+if [[ $continue_prompt = 'y' ]]; then
```

Comment:
I'd consider flipping this logic and just exiting it `$continue_prompt` does _not_ equal `y` – then you can avoid indenting the whole block.